data/json/mapgen/dairy_farm.json

```diff
@@ -0,0 +1,251 @@
+[
+  {
+    "type":"monstergroup",
+    "name" : "GROUP_COWS",
+    "is_safe" : true,
+    "default" : "mon_null",
+    "monsters":[{"monster" : "mon_cow", "cost_multiplier" : 10, "freq": 100, "ends" : 168 },
+    {"monster" : "mon_cow", "freq": 50, "cost_multiplier" : 5, "starts" :  336  }
+    ]
```

Comment:
Spawn frequency is rate on a 1000 point scale, not 100.

The default mon, mon_null is chosen for all cases wherein a one-thousand sided die is rolled and 1-100 is  not chosen.  thus 100 points of freq is 10%